### PR TITLE
Update node-version

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
Dont merge yet!

## Upgrade to Node 18

Before merging 

- [x] update node version in netlify
- [x] update node version in release action
- [x] test properly

### CODE REVIEW

File: src/ui/layouts/connectors/ServicesSelectorComponent/index.module.scss [hunk 1]
```diff
@@ -1,5 +1,7 @@
-img {
-  height: 20px;
+.service_selectro_wrapper {
```
Comment (Cahllagerfeld) [inline src/ui/layouts/connectors/ServicesSelectorComponent/index.module.scss:1]: todo:fix typo

**Review by Cahllagerfeld — COMMENTED** [review]
Also fixes a bug related to img styling

**Comment by talhainvenxion** [comment]
@Cahllagerfeld when you are done with then please share the node version. I'll update it on netlify.

**Comment by talhainvenxion** [comment]
@Cahllagerfeld, I've tested it on Netlify, and it's working perfectly. We're all set to merge this PR.

**Pull request merged**